### PR TITLE
Remove Travis examples and documentation

### PR DESCRIPTION
**Pull request by davidbuzinski**
* Remove `.travis.yml`
* Remove references to travis in README
* Update versions in Azure pipeline comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -306,7 +289,6 @@ matlab-test:
 
 
 ## Caveats
-* MATLAB builds on Travis CI are available only for public projects.
 * MATLAB builds on Azure DevOps, CircleCI, and GitHub Actions that use CI service-hosted agents are also available only for public projects. However, these integrations can also be used in private projects that leverage self-hosted runners/agents.
```
Comment (mw-hrastega) [inline README.md:292]: This  sounds outdated.

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (davidbuzinski) [reply README.md:292]: Do you think we should remove the entire section or add some version of the new "Licensing" information that we incorporated into the Azure doc recently?

Comment (mw-hrastega) [reply README.md:292]: It depends on how important the information is. We can add something along these lines if useful (not sure if it applies to GitLab CI/CD):

On cloud-hosted agents, you need a [MATLAB batch licensing token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token) if your project is private or if your pipeline includes transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;. You can request a token by contacting MathWorks&reg; at [batch-tokens@mathworks.com](mailto:batch-tokens@mathworks.com).

Comment (davidbuzinski) [reply README.md]: Good suggestion. You're right in thinking that it doesn't currently apply to GitLab CI/CD or the hosted CI services. I have tried to modify the language to point this out. Let me know if you have a better suggestion or want some changes to the language.

**Pull request merged**